_posts/2021-03-01-changelog-142.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #142"
+date: 2021-03-01 17:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6877 -->
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6835 -->
+* Debugging is now available in [PyCharm Professional](https://www.jetbrains.com/pycharm/) and
+  [GoLand](https://www.jetbrains.com/go/) using
```

Comment:
Note, we've already announced debugger support in GoLand in previous changelog.  
My suggestion for this item:
- drop info about GoLand
- add note that it's availbale only since 2021.1
- extract separate item that now the plugin suggest to install `Native Debugging Support` plugin if you try to run debug process in PyCharm Pro and GoLand